src/i18n/en.json

```diff
@@ -0,0 +1,138 @@
+  },
+  "reports": {
+    "allow_enable_trial_HTML": "<p>Activate the trial period and access detailed Reports on the behavior of users inside your Website or E-commerce. Discover which are the most visited pages, how many visitors have an Email identified by Doppler and how many don't. Any doubts? Press <a target=\"_blank\" href=\"https://help.fromdoppler.com/en/advanced-reports\">HELP.</a></p>",
+    "allow_enable_trial_button": "Start the free trial",
+    "allow_enable_trial_title": "Try On-Site Tracking Automation for a limited time",
+    "datahub_not_active_HTML": "<p>How do users behave on your Website or E-commerce? Track their behavior for a period of time and find out which are the most visited pages, how many of those visits have an Email identified by Doppler and how many don’t. By tracking the user’s journey you'll be able to detect vanishing points and opportunities for improvement!</p><p>You haven’t enabled On-Site Tracking yet. You can trigger it from the <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking</a> option in the Control Panel.</p>",
+    "datahub_not_active_title":"Track user behavior and optimize your Marketing actions",
+    "datahub_not_domains_title":"Add your web domain and analyze the behavior of your users",
+    "no_domains_HTML": "<p>Register the domain (s) you want to track and access to detailed Reports. Discover which are the most visited pages of your Website or E-commerce, how many visitors have been identified by Doppler and how many have not. Any doubts? Press <a target=\"_blank\" href=\"https://help.fromdoppler.com/en/advanced-reports\">HELP.</a></p><p><a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">Add your domain</a></p>",
+    "upgrade_account_free_HTML": "<p>Upgrade your account and access detailed Reports on the behavior of users on your Website or E-commerce. Discover which are the most visited pages, how many visitors have an Email that Doppler has identified and how many don't. Any doubts? Press <a target=\"_blank\" href=\"https://help.fromdoppler.com/en/advanced-reports\">HELP.</a></p><p>By joining any Paid Plan you can enjoy FOR FREE this feature. Limited time only. <a href=\"{dopplerBaseUrl}/ControlPanel/AccountPreferences/UpgradeAccount?Plan=monthly\">UPGRADE NOW.</a></p>",
+    "upgrade_account_free_title": "Analyze your Subscriber's behaviour and improve your strategy"
+  },
+  "reports_box": {
+    "to": "to",
+    "visits_description_with_email": "Total number of users who visited your Website and whose Email has been identified by Doppler. If a user entered several times, only one will be counted.",
+    "visits_description_without_emails": "Total number of users who visited your Website and whose Email has not been identified by Doppler. If a user entered several times, only one will be counted.",
+    "visits_with_email": "Users with emails",
```

Comment:
`Emails`? same in the line bellow `Email`

Reply:
@carodipietro esto que comenta @cbernat no es parte de los cambios de @GusBaamonde, pero me parece que los comentarios de Cecilia son correctos, ¿Crees que deberíamos aplicar esta sugerencia?

Reply:
In my opinion it should be Email in singular in both cases.

Reply:
`Users with Email`